e2e/features/CreateUserRole.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Join Group
+  In order to join a group
+  As a user
+  I want to search a group and join it
+  Background:
+    Given I'm in the homepage
+    And I log in as "default1" with password "password"
+    Then The "Register" menu is not present
+
+  Scenario: Join a group
+    Given I am in group-list page "1"
+    Then I click the "Join" button
```

Comment:
Clicking the "Join" button should be a When, not a Then. After joining, there should be a new Then step that checks the expected outcome, for instance, a confirmation message or the user appearing as one of the members of the group.

Reply:
Fixed it, do I have to create another pull request?

Reply:
No need to open another pull request, changes are added automatically to the open one if they are made to the same branch